cmd/terramate/cli/run.go

```diff
@@ -421,3 +434,64 @@ func (c *cli) loadAllStackEnvs(runs []runContext) (map[prj.Path]runutil.EnvVars,
 	}
 	return stackEnvs, nil
 }
+
+func (c *cli) createCloudPreview(runs []runContext) map[string]string {
+	previewRuns := make([]previews.RunContext, len(runs))
+	for i, run := range runs {
+		previewRuns[i] = previews.RunContext{
+			Stack: run.Stack,
+			Cmd:   run.Cmd,
+		}
+	}
+
+	affectedStacks, err := c.computeSelectedStacks(false, true, cloudstack.NoFilter)
+	if err != nil {
+		fatal("computing affected stacks", err)
+	}
+
+	affectedStacksMap := map[string]*config.Stack{}
+	for _, st := range affectedStacks {
+		affectedStacksMap[st.Stack.ID] = st.Stack
+	}
+
+	githubEventPath, ok := os.LookupEnv("GITHUB_EVENT_PATH")
+	if !ok {
+		fatal("env var GITHUB_EVENT_PATH not found, not generating previews", nil)
+	}
+
+	prUpdatedAt := time.Now().UTC().Unix()
+	if eventPRUpdatedAt := github.GetEventPRUpdatedAt(githubEventPath); eventPRUpdatedAt != nil {
```

Comment:
it would be nice if we can have a testing env var for injecting the updated at.
```
$ make build/test
$ TM_TEST_PREVIEW_UPDATED_AT=... ./bin/terramate ...
```

Reply:
`TM_IGNORE_EVENT_PATH`